testBuild_Data/output_log.txt

```diff
@@ -15,7 +15,7 @@ Platform assembly: B:\ProgramScratch\Swipe Warriors\SwipeWarriors\testBuild_Data
 Loading B:\ProgramScratch\Swipe Warriors\SwipeWarriors\testBuild_Data\Managed\UnityEngine.UI.dll into Unity Child Domain
 Platform assembly: B:\ProgramScratch\Swipe Warriors\SwipeWarriors\testBuild_Data\Managed\UnityEngine.Networking.dll (this message is harmless)
 Loading B:\ProgramScratch\Swipe Warriors\SwipeWarriors\testBuild_Data\Managed\UnityEngine.Networking.dll into Unity Child Domain
-- Completed reload, in  0.100 seconds

```

Comment:
Hmm, this testBuild_Data directory shouldn't be here, because I've written the .gitignore to ignore the builds. Can you try manually deleting the directory before pull requesting? don't worry it wouldnt break the game